src/olympia/scanners/migrations/0023_auto_20200122_1649.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.9 on 2020-01-22 16:49
+
+from django.db import migrations
+
+
+def create_waffle_switch(apps, schema_editor):
```

Comment:
Do we really need the waffle switch if it's gated by a permission check too?  What's the scenario where we want to turn it off and just telling the handful of users with that permission not to access it isn't sufficient?

Reply:
Yes, we do because we need a confirmation from secops as well (mostly to know whether we need extra protection)